types/contract-proxy-kit/index.d.ts

```diff
@@ -0,0 +1,63 @@
+// Type definitions for contract-proxy-kit 1.0
+// Project: https://github.com/gnosis/contract-proxy-kit#readme
+// Definitions by: Alan Lu <https://github.com/cag>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.6
+
+import { ethers } from 'ethers';
+
+export = CPK;
+
+interface NetworkConfigEntry {
```

Comment:
If you want your users to be able to import these types from the module, you should wrap them in a namespace called `CPK`.